Singer.API/Controllers/CareUserController.cs

```diff
@@ -8,107 +8,43 @@
 using Singer.DTOs;
 using Singer.Models.Users;
 using Singer.Services;
+using Singer.Helpers.Exceptions;
 
 namespace Singer.Controllers
 {
    [Route("api/[controller]")]
-   [Authorize()]
-   public class CareUserController : Controller
+   //[Authorize()]
+   public class CareUserController : DataControllerBase<CareUser, CareUserDTO, CreateCareUserDTO, UpdateCareUserDTO>
    {
-      private readonly CareUserService _userService;
-      private readonly IMapper _mapper;
-
-      public CareUserController(CareUserService service, IMapper mapper)
+      private readonly CareUserService careUserService;
+      public CareUserController(CareUserService databaseService) : base(databaseService)
```

Comment:
I would create an interface of the CareUserService and use that for the dependency injection